docs/install.md

```diff
@@ -63,18 +63,18 @@ scoop install helm
 ### From Script
 
 Helm now has an installer script that will automatically grab the latest version
-of the Helm client and [install it locally](https://raw.githubusercontent.com/helm/helm/master/scripts/get).
+of the Helm client and [install it locally](https://git.io/get_helm.sh).
 
 You can fetch that script, and then execute it locally. It's well documented so
 that you can read through it and understand what it is doing before you run it.
 
 ```
-$ curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+$ curl -LO https://git.io/get_helm.sh
 $ chmod 700 get_helm.sh
 $ ./get_helm.sh
 ```
 
-Yes, you can `curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash` that if you want to live on the edge.
+Yes, you can `curl https://git.io/get_helm.sh | bash` that if you want to live on the edge.
```

Comment:
This would have to change to `curl -L https://git.io/get_helm.sh | bash` for this to work IIRC. Missing the `-L` flag to follow redirects

Reply:
nice catch!